src/unix/fs.c

```diff
@@ -254,6 +254,37 @@ static ssize_t uv__fs_futime(uv_fs_t* req) {
 }
 
 
+static ssize_t uv__fs_lutime(uv_fs_t* req) {
+#if defined(__linux__)                                                         \
+    || defined(_AIX71)                                                         \
+    || defined(__sun)                                                          \
+    || defined(__HAIKU__)
+  struct timespec ts[2];
+  ts[0].tv_sec  = req->atime;
+  ts[0].tv_nsec = (uint64_t)(req->atime * 1000000) % 1000000 * 1000;
+  ts[1].tv_sec  = req->mtime;
+  ts[1].tv_nsec = (uint64_t)(req->mtime * 1000000) % 1000000 * 1000;
```

Comment:
Maybe now would be a good time to DRY this and move it into a function because I count at least 4 other instances in src/unix/fs.c:
```c
static struct timespec uv__fs_to_timespec(double time) {
  struct timespec ts;
  ts.tv_sec  = time;
  ts.tv_nsec = (uint64_t)(time * 1000000) % 1000000 * 1000;
  return ts;
}

static ssize_t uv__fs_lutime(uv_fs_t* req) {
  // ...
  ts[0] = uv__fs_to_timespec(req->atime);
  ts[1] = uv__fs_to_timespec(req->mtime);
  // ...
}
```
And ditto for `struct timeval`, of course. :-)